src/main/java/io/gingersnapproject/cdc/configuration/Configuration.java

```diff
@@ -3,11 +3,15 @@
 import java.util.Map;
 
 import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
 import io.smallrye.config.WithName;
 
 @ConfigMapping(prefix = "gingersnap")
 public interface Configuration {
 
+   @WithDefault("false")
+   boolean dynamicMembership();
```

Comment:
looks like every property that starts with `gingersnap` must be mapped, or it throws an exception in the start-up.